samples/DevTestLabs/GatewaySample/arm/gateway/Readme.md

```diff
@@ -0,0 +1,25 @@
+# Create a Token Authentication Based Remote Desktop Gateway
```

Comment:
It doesn't look like this file's content matches that of the folder. Please update accordingly.

Reply:
done.